thinkhazard/tests/views/__init__.py

```diff
@@ -82,89 +79,174 @@ def populate_db():
     ])
     geometry = from_shape(shape, 4326)
 
-    div_level_1 = AdministrativeDivision(**{
+    # admin_div_10 is a country (division level 1)
+    admin_div_10 = AdministrativeDivision(**{
         'code': 10,
         'leveltype_id': 1,
         'name': u'Division level 1'
     })
-    div_level_1.geom = geometry
-    DBSession.add(div_level_1)
+    admin_div_10.geom = geometry
+    DBSession.add(admin_div_10)
 
-    div_level_1_2 = AdministrativeDivision(**{
+    # admin_div_11 is another country (division level 1)
+    admin_div_11 = AdministrativeDivision(**{
         'code': 11,
         'leveltype_id': 1,
         'name': u'Division level 1 2'
     })
-    div_level_1_2.geom = geometry
-    DBSession.add(div_level_1_2)
+    admin_div_11.geom = geometry
+    DBSession.add(admin_div_11)
+
+    # admin_div_12 is another country (division level 1)
+    admin_div_12 = AdministrativeDivision(**{
+        'code': 12,
+        'leveltype_id': 1,
+        'name': u'Division level 1 3'
+    })
+    admin_div_12.geom = geometry
+    DBSession.add(admin_div_12)
+
+    # admin_div_13 is another country (division level 1)
+    admin_div_13 = AdministrativeDivision(**{
+        'code': 13,
+        'leveltype_id': 1,
+        'name': u'Division level 1 4'
+    })
+    admin_div_13.geom = geometry
+    DBSession.add(admin_div_13)
 
-    div_level_2 = AdministrativeDivision(**{
+    # admin_div_20 is a region (division level 2)
+    # it's parent is admin_div_10
+    admin_div_20 = AdministrativeDivision(**{
         'code': 20,
         'leveltype_id': 2,
         'name': u'Division level 2'
     })
-    div_level_2.parent_code = div_level_1.code
-    div_level_2.geom = geometry
-    DBSession.add(div_level_2)
+    admin_div_20.parent_code = admin_div_10.code
+    admin_div_20.geom = geometry
+    DBSession.add(admin_div_20)
 
     shape = MultiPolygon([
         Polygon([(0, 0), (0, 1), (.5, 1), (.5, 0), (0, 0)])
     ])
     geometry = from_shape(shape, 4326)
 
-    div_level_3_1 = AdministrativeDivision(**{
+    # admin_div_31 is a department (division level 3)
```

Comment:
I think divisions are either country, province or region (ie. not department).
https://github.com/GFDRR/thinkhazard/blob/master/thinkhazard/scripts/initializedb.py#L101